cmd/gateway-azure.go

```diff
@@ -655,7 +655,7 @@ func (a *azureObjects) PutObjectPart(bucket, object, uploadID string, partID int
 	}
 
 	info.PartNumber = partID
-	info.ETag = etag
+	info.ETag = azureToS3ETag(etag)
```

Comment:
You could fix the line 627 like `etag = getMD5Hash([]byte(azureToS3ETag(mustGetUUID())))` and rest are taken care automatically.

Reply:
@balamurugana you mean `etag = azureToS3ETag(getMD5Hash([]byte(mustGetUUID())))`

Reply:
@krishnasrinivas no. my proposal doesn't consume extra place in BlockID string

Reply:
> etag = azureToS3ETag(getMD5Hash([]byte(mustGetUUID())))
Also this would be the only change needed, other changes are not needed

Reply:
Done